core/src/main/java/org/pentaho/di/connections/vfs/BaseVFSConnectionDetails.java

```diff
@@ -52,6 +55,34 @@ public Map<String, String> getProperties() {
     return props;
   }
 
+  /**
+   * Gets if the VFS connection supports root path or not.
+   * @returns {@code true} if VFS connection supports root path; {@code false} otherwise.
+   * @default {@code true}
+   */
+  @Override
+  public boolean isSupportsRootPath() {
+    return true;
+  }
+
+  /**
+   * Gets the root path of a VFS Connection.
+   * returns the root path.
+   */
+  @Override
+  public String getRootPath() {
+    return rootPath;
+  }
+
+  /**
+   * Sets the root path, given as a string
+   * @param rootPath The root path
+   */
```

Comment:
If there's nothing different in this doclet compared to the interface, please, remove it.

Reply:
changes addressed.